etc/helm/pachyderm/values.yaml

```diff
@@ -554,6 +554,14 @@ pachd:
     # deploy.
     create: true
 
+kubeEventTail:
+  # Deploys a lightweight app that watches kubernetes events and echos them to logs.
+  enabled: false
```

Comment:
Let's default to true.  That will make it deploy during CI, and will tell you if you added the magic security context settings :)